homework/index.js

```diff
@@ -0,0 +1,64 @@
+function getPromise() {
```

Comment:
When naming a function, try to specify what it really does (and if it does too much; split it up). In this case: get *what* promise?